core/plugins/org.polarsys.capella.core.semantic.queries/src/org/polarsys/capella/core/semantic/queries/basic/queries/OperationalActivityInvolvingCapabilities.java

```diff
@@ -0,0 +1,63 @@
+/*******************************************************************************
+ * Copyright (c) 2022 THALES GLOBAL SERVICES.
```

Comment:
Copyright (c) 2022 Obeo.

contributors - summary/scope of the change/ticket id